utils/utils.go

```diff
@@ -176,3 +192,38 @@ func RandomString(strlen int) string {
 	}
 	return string(result)
 }
+
+// Contains evaluates whether s contains x.
+func Contains(s []string, x string) bool {
+	for _, n := range s {
+		if x == n {
+			return true
+		}
+	}
+	return false
+}
+
+func RemoveFileInfo(fileInfo os.FileInfo, fileInfoName string) error {
+	if fileInfo.IsDir() {
```

Comment:
Why can't you just use [os.RemoveAll(dirOrFileName)](https://godoc.org/os#RemoveAll)? Not sure it makes sense to have this method at all...